Source/WebKit/UIProcess/API/Cocoa/WKNavigationResponse.mm

```diff
@@ -39,7 +39,8 @@ - (void)dealloc
     if (WebCoreObjCScheduleDeallocateOnMainRunLoop(WKNavigationResponse.class, self))
         return;
 
-    _navigationResponse->~NavigationResponse();
+    // FIXME: This RefPtr should not be necessary. Remove it once clang static analyzer is fixed.
+    RefPtr { _navigationResponse.get() }->~NavigationResponse();
```

Comment:
There should be a specific bug on file that is referenced from the FIXMEs, for each such unique analyzer issue, ideally assigned to @rniwa.

Reply:
TBH I copied this from a different place in the same file, so I assumed this is a known issue. @rniwa ?